src/terminal7.js

```diff
@@ -313,17 +315,13 @@ echo "${fp}" >> ~/.config/webexec/authorized_fingerprints`
         // peerbook button and modal
         modal = document.getElementById("peerbook-modal")
         modal.querySelector(".close").addEventListener('click',
-            ev => this.clear() )
-        modal.querySelector(".save").addEventListener('click',
-            ev => {
-                this.setPeerbook()
-                this.clear()
-            })
+            () => this.clear() )
         document.getElementById('add-peerbook').addEventListener(
-            'click', ev => {
+            'click', () => {
                 this.logDisplay(false)
                 // modal.querySelector("form").reset()
-                document.getElementById("peerbook-modal").classList.remove("hidden")
+                modal.classList.remove("hidden")
+                this.setPeerbook()
```

Comment:
I don't like the function name. `set` prefix is usually used to set a database value and almost never to query user for input. How about `peerbookForm` ?